src/NServiceBus.Transport.Msmq/ReceiveStrategy.cs

```diff
@@ -123,7 +123,7 @@ protected async Task<ErrorHandleResult> HandleError(Message message, Dictionary<
             }
             catch (Exception ex)
             {
-                criticalError.Raise($"Failed to execute recoverability actions for message `{message.Id}`", ex);
+                criticalError.Raise($"Failed to execute recoverability policy for message with native ID: `{message.Id}`", ex);
```

Comment:
Is it **always** native ID though?

Reply:
Yes it should be https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/Transports/MessageContext.cs#L43

The transport can choose to reuse the nsb id but it would still be the "native id that the transport choose to use"

Makes sense?